### ISSUE TITLE
QueuePool limit reached

### ISSUE THREAD

**Issue by indy-independence** [SUNET/cnaas-nms#188]
**Describe the bug**
SQLalchemy QueuePool limit reached

**To Reproduce**
1. Have 50+ devices and/or slow database?
2. Sync all devices
3. Exception raised

**Expected behavior**
No exception should be raised, just slow down

**Outputs**
```
  File "./cnaas_nms/confpush/sync_devices.py", line 339, in push_sync_device
    dev: Device = session.query(Device).filter(Device.hostname == hostname).one()
...
sqlalchemy.exc.TimeoutError: QueuePool limit of size 50 overflow 0 reached, connection timed out, timeout 30 (Background on this error at: http://sqlalche.me/e/13/3o7r)
```

**Environment:**
 - Setup: docker
 - Version: v1.2.1
 - Client: webui

**Additional context**
Quite big job db?

**Comment by indy-independence** [comment]
Maybe just change:
cnaas_nms/db/session.py:engine = create_engine(conn_str, pool_size=50, max_overflow=0)
to:
max_overflow=50

**Comment by indy-independence** [comment]
Fix in #190
